include/ck/utility/type_convert.hpp

```diff
@@ -109,9 +109,10 @@ inline __host__ __device__ f8_t f8_convert_sr<f8_t, float>(float x)
 {
     constexpr int seed = 42;
     uint32_t rng       = prand_generator<float, seed>(reinterpret_cast<uintptr_t>(&x), x);
+    float max_fp8      = 240.0f;
+    float inf          = std::numeric_limits<float>::infinity();
+    x = x > max_fp8 && x < inf ? max_fp8 : (x > -inf && x < -max_fp8 ? -max_fp8 : x);
```

Comment:
Can't we use here [`std::isinf`](https://en.cppreference.com/w/cpp/numeric/math/isinf) and use `if (isinf(x))....` The code would be way more readable.

Reply:
OK, will change to 
if(!std::isinf(x)) x = x > max_fp8 ? max_fp8 : (x < -max_fp8 ? -max_fp8 : x);
Sounds good?